test/cpp/jit/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ set(JIT_TEST_SRCS
   ${JIT_TEST_ROOT}/test_lite_trainer.cpp
   ${JIT_TEST_ROOT}/test_memory_dag.cpp
   ${JIT_TEST_ROOT}/test_misc.cpp
+  ${JIT_TEST_ROOT}/test_profiling_pass.cpp
```

Comment:
nit: can you also add this to the internal buck file?

(also... it's not alphabetically sorted)